projects/ngneat/transloco/src/lib/tests/transloco.transpiler.spec.ts

```diff
@@ -86,30 +108,42 @@ describe('TranslocoTranspiler', () => {
         lang: 'lang',
         nes: { ted: 'supporting nested values!' }
       });
-      const parsed = parser.transpile('Hello {{ withKeys }} {{ from }} {{ lang }} {{nes.ted}}', {}, lang);
+      const parsed = parser.transpile(
+        `Hello ${interpolationMarkings[0]} withKeys ${interpolationMarkings[1]} ${interpolationMarkings[0]} from ${interpolationMarkings[1]} ${interpolationMarkings[0]} lang ${interpolationMarkings[1]} ${interpolationMarkings[0]}nes.ted${interpolationMarkings[1]}`,
+        {},
+        lang
+      );
       expect(parsed).toEqual('Hello with keys from lang supporting nested values!');
     });
 
     it('should translate simple string with from lang with nested params', () => {
       const lang = flatten({
-        dear: 'dear {{name}}',
-        hello: 'Hello {{dear}}'
+        dear: `dear ${interpolationMarkings[0]}name${interpolationMarkings[1]}`,
```

Comment:
create a function called `wrapParam` which replaces: `${interpolationMarkings[0]}paramName${interpolationMarkings[1]}` and replace all instances with the function call.